src/ucs/memory/rcache.c

```diff
@@ -145,6 +181,27 @@ void ucs_rcache_region_log(const char *file, int line, const char *function,
                      region_desc);
 }
 
+void ucs_rcache_set_params(const ucs_rcache_config_t *rcache_config,
+                           ucs_rcache_params_t *rcache_params)
+{
+    if (rcache_config != NULL) {
```

Comment:
can we set default values for **all** fields in rcache_params?

Reply:
@yosefe Just to clarify -- your suggestion is to always set default values for all fields and then have that be overwritten. Is that correct? If yes, does it make sense to change the function name to ucs_rcache_set_default_params?